src/pipemode_op/RecordReader/TextLineRecordReader.cpp

```diff
@@ -55,7 +57,6 @@ bool TextLineRecordReader::ReadRecord(std::string* data) {
             if (data->size() == 0) {
                 return false;
             } else {
-                data->shrink_to_fit();
```

Comment:
why remove this call? Is it deprecated or no longer required?

Reply:
tensorflow::tstring does not implement shrink_to_fit(), re-implemented as resize to it's string size